src/main/kotlin/org/jitsi/videobridge/transport/ice/IceTransport.kt

```diff
@@ -215,28 +215,19 @@ class IceTransport @JvmOverloads constructor(
 
     /**
      * Send data out via this transport
-     *
-     * TODO: should we ensure that we're running before sending?
      */
     fun send(data: ByteArray, off: Int, length: Int) {
-        try {
-            iceComponent.socket.send(DatagramPacket(data, off, length))
-            packetStats.numPacketsSent++
-        } catch (e: IOException) {
-            logger.error("Error sending packet", e)
-            throw RuntimeException()
+        if (running.get()) {
+            try {
+                iceComponent.socket.send(DatagramPacket(data, off, length))
+                packetStats.numPacketsSent++
+            } catch (e: IOException) {
+                logger.error("Error sending packet", e)
+                throw RuntimeException()
+            }

```

Comment:
You just ignore the send entirely if you're not running - do you want to accumulate a stat or something?